python/ray/dag/tests/experimental/test_accelerated_dag.py

```diff
@@ -1261,6 +1261,30 @@ def test_driver_and_actor_as_readers(ray_start_cluster):
         dag.experimental_compile()
 
 
+def test_payload_too_large(ray_start_regular):
```

Comment:
nit: `test_large_payload`, it is not too large since we are supporting it

Reply:
Done